packages/manager/src/components/TextField.tsx

```diff
@@ -64,25 +65,69 @@ const useStyles = makeStyles((theme: Theme) => ({
 }));
 
 interface BaseProps {
-  className?: any;
+  /**
+   * className to apply to the underlying TextField component
+   */
+  className?: string;
+  /**
+   * Data attributes are applied to the underlying TextField component for testing purposes
+   */
   dataAttrs?: Record<string, any>;
+  /**
+   * Applies editable styles
+   */
   editable?: boolean;
+  /**
+   * Adds error grouping to TextField
+   */
   errorGroup?: string;
+  /**
+   * When defined, makes the input show an error state with the defined text
+   */
   errorText?: string;
+  /**
+   * Makes the TextField use 100% of the available width
+   */
   expand?: boolean;
+  /**
+   * Makes the error text have the absolute positioning
+   */
   hasAbsoluteError?: boolean;
+  /**
+   * Placement of the `helperText`
+   */
   helperTextPosition?: 'top' | 'bottom';
+  /**
+   * Hides the `label`
+   */
   hideLabel?: boolean;
+  /**
+   * Allows you to manually set an htmlFor input id. One will automaticly be generated by the `label` if this is unset
+   */
   inputId?: string;
+  /**
+   * Displays a loading spinner at the end of the Text Field
+   */
   loading?: boolean;
   /**
-   * The number amounts allowed in TextField and
-   * the "type" prop must also be set to "number"
+   * The maximum number allowed in TextField. The "type" prop must also be set to `number`
    */
   max?: number;
+  /**
+   * The minimum number allowed in TextField. The "type" prop must also be set to `number`
+   */
   min?: number;
+  /**
+   * Removed the default top margin (16px)
```

Comment:
```suggestion
   * Removes the default top margin (16px)
```
Keeps present tense consistent with the other props